test/feature/reverse_routing.feature

```diff
@@ -0,0 +1,31 @@
+Feature: reverse routing
+
+  Background:
+    Given an empty "controllers" directory
+
+
+  Scenario: routes can define reverse routes with 'as'
+    Given a file "routes.coffee" with the contents
+      """
+      module.exports = ({ GET }) ->
```

Comment:
This is a good example for where providing the whole routes file makes less sense. What we really care about here is just the `GET` directive. I think the spec should just contain that.

```cucumber
Scenario: Providing a custom name for routing helpers
  Given the routing definition
    """
    GET '/some/route', to: 'test#index', as: 'foobar'
    """
  Then I have a routing helper "res.locals.paths.foobar" that returns "/some/route"
```